src/iwant_bot/db.py

```diff
@@ -52,7 +52,8 @@ def save_message(self, message, nickname):
         session = self._make_session()
         try:
             query = session.query(
-                sqlalchemy.exists().where(Nickname.name == nickname))
+                sqlalchemy.exists().where(Nickname.name == nickname)
+                )
```

Comment:
@chauffer I found all three possibilities in PEP8, even the ) at the end of first line. Please, if you mention an internal agreement, can you be more specific?